src/replica.rs

```diff
@@ -293,9 +295,12 @@ pub struct ReplicaSpec<K: KVStore, A: App<K> + 'static, N: Network + 'static> {
     #[builder(default, setter(transform = |handler: impl Fn(&UpdateHighestQCEvent) + Send + 'static| Some(Box::new(handler) as HandlerPtr<UpdateHighestQCEvent>),
     doc = "Register a handler closure to be invoked after the replica updates its highest QC. Optional."))]
     on_update_highest_qc: Option<HandlerPtr<UpdateHighestQCEvent>>,
-    #[builder(default, setter(transform = |handler: impl Fn(&UpdateLockedViewEvent) + Send + 'static| Some(Box::new(handler) as HandlerPtr<UpdateLockedViewEvent>),
-    doc = "Register a handler closure to be invoked after the replica updates its locked view. Optional."))]
-    on_update_locked_view: Option<HandlerPtr<UpdateLockedViewEvent>>,
+    #[builder(default, setter(transform = |handler: impl Fn(&UpdateLockedQCEvent) + Send + 'static| Some(Box::new(handler) as HandlerPtr<UpdateLockedQCEvent>),
+    doc = "Register a handler closure to be invoked after the replica updates its locked QC. Optional."))]
+    on_update_locked_view: Option<HandlerPtr<UpdateLockedQCEvent>>,
```

Comment:
This function's name seems outdated since we've now moved from `locked_view` to `locked_qc`.